tests/Oryx.Integration.Tests/Php/PhpWordPressTest.cs

```diff
@@ -68,5 +68,128 @@ public async Task WordPress51(string phpVersion)
+                async (hostPort) =>
+                {
+                    // this is to test wordpress and mysql connection is working
+                    var testdbconnectiondata = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/testdb.php");
+                    Assert.DoesNotContain("Unable to connect to MySQL",testdbconnectiondata);
+                    Assert.Contains("Connected successfully", testdbconnectiondata);
+
+                    // this is to test regular wordpress site is working
+                    var data = await _httpClient.GetStringAsync($"http://localhost:{hostPort}/wp-login.php");
+                    Assert.Contains("Powered by WordPress", data);
+                    Assert.Contains("Remember Me", data);
+                    Assert.Contains("Lost your password?", data);
+                    Assert.Contains("Back to localsite", data);
+                });
+        }
+
+        [Theory]
+        [InlineData("7.4")]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
+        public async Task CanBuildAndRun_Wordpress_SampleApp_With_Phpfpm(string phpVersion)
+        {
+            // Arrange
+            var appName = "wordpress-example";
+            var hostDir = Path.Combine(_hostSamplesDir, "php", appName);
+            var volume = DockerVolume.CreateMirror(hostDir);
+            var appDir = volume.ContainerDir;
+            var phpimageVersion = string.Concat(phpVersion, "-fpm");
+
+            // build-script to download wordpress cli and build
+            var buildScript = new ShellScriptBuilder()
+                .AddCommand($"cd {appDir}")
+                .AddCommand("curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar")
+                .AddCommand("chmod +x wp-cli.phar")
+                .AddCommand("./wp-cli.phar core download")
+                .AddCommand($"oryx build {appDir} --platform {PhpConstants.PlatformName} --language-version {phpVersion}")
+                .AddDirectoryExistsCheck("wp-admin")
+                .AddFileDoesNotExistCheck("wp-config.php")
+                .ToString();
+
+            // run script to finish wordpress configuration and run the app
+            var runScript = new ShellScriptBuilder()
+                .AddCommand($"cd {appDir}")
+                .AddFileExistsCheck("oryx-manifest.toml")
+                .AddFileExistsCheck("wp-cli.phar")
+                .AddFileExistsCheck("create_wordpress_db.sh")
+                .AddFileExistsCheck("configure_wordpress.sh")
+                .AddCommand($"chmod +x create_wordpress_db.sh && ./create_wordpress_db.sh")
+                .AddCommand($"chmod +x configure_wordpress.sh && ./configure_wordpress.sh")
+                .AddCommand($"oryx create-script -appPath {appDir} -bindPort {ContainerPort} -output {RunScriptPath}")
+                .AddCommand("mkdir -p /home/site/wwwroot")
+                .AddCommand($"cp -a {appDir}/. /home/site/wwwroot")
```

Comment:
will `cp -rf` be better instead of `cp -a`?

Reply:
I tried with cp -rf first and it was failing so trying with cp -a, as per doc cp -a seems to be a better version of cp -rf

Reply:
gotcha, thanks